docs/design/coreclr/botr/porting-ryujit.md

```diff
@@ -1,37 +1,97 @@
 # RyuJIT: Porting to different platforms
 
-## What is a Platform?
-* Target instruction set and pointer size
-* Target calling convention
+First, understand the JIT architecture by reading [RyuJIT overview](ryujit-overview.md).
+
+# What is a Platform?
+
+* Target instruction set
+* Target pointer size
+* Target operating system
+* Target calling convention and ABI (Application Binary Interface)
 * Runtime data structures (not really covered here)
 * GC encoding
-  * So far only JIT32_GCENCODER and everything else
-* Debug info (so far mostly the same for all targets?)
-* EH info (not really covered here)
+  * So far only JIT32_GCENCODER (used by Windows x86) and everything else
+* Debug information (so far mostly the same for all targets?)
+* EH (exception handling) information (not really covered here)
 
-One advantage of the CLR is that the VM (mostly) hides the (non-ABI) OS differences
+One advantage of the CLR is that the VM (mostly) hides the (non-ABI) OS differences.
 
-## The Very High Level View
-* 32 vs. 64 bits
-  * This work is not yet complete in the backend, but should be sharable
+# The Very High Level View
+
+The following components need to be updated, or target-specific versions created, for a new platform.
+
+* The basics
+  * target.h
 * Instruction set architecture:
+  * registerXXX.h
+  * emitXXX.h, emitfmtXXX.h
   * instrsXXX.h, emitXXX.cpp and targetXXX.cpp
   * lowerXXX.cpp
-  * codeGenXXX.cpp and simdcodegenXXX.cpp
+  * lsraXXX.cpp
+  * codegenXXX.cpp and simdcodegenXXX.cpp
   * unwindXXX.cpp
-* Calling Convention: all over the place
+* Calling Convention and ABI: all over the place
+* 32 vs. 64 bits
+  * Also all over the place. Some pointer size-specific data is centralized in target.h, but probably not 100%.
+
+# Porting stages and steps
+
+There are several steps to follow to port the JIT (some of which should be parallelized):
```

Comment:
```suggestion
There are several steps to follow to port the JIT (some of which can be parallelized):
```
?

Reply:
Should this be "some of which can be done in parallel"?